### PR TITLE
font weight fixed - iOS back to 15.2

### PR DESCRIPTION
**Pull request by detoti**
## Describe your changes 

- font weight fixed to work on iOS 15.2
- iOS version back to 15.2
 
## Checklist before requesting a review

Put an "X" in the boxes that apply.

- [X] I have performed a self-review of my code
- [X] New and existing unit tests pass locally with my changes
- [X] Does not contain commented code

### CODE REVIEW

**Comment by denise-techlead** [comment]
Great! Please wait a moment while I review your code. 👀

**Comment by denise-techlead** [comment]
The unit tests did not pass. This means there is something wrong with your solution. Fix it and create a new commit on this branch to update this Pull Request. 👀

**Comment by denise-techlead** [comment]
The unit tests did not pass. This means there is something wrong with your solution. Fix it and create a new commit on this branch to update this Pull Request. 👀

File: EvercodeApp/SplashView.swift [hunk 1]
```diff
@@ -2,7 +2,17 @@ import SwiftUI
 
 struct SplashView: View {
     var body: some View {
-        Text("Welcome to Evercode!")
+        VStack {
```
Comment (rdgborges) [inline EvercodeApp/SplashView.swift:5]: Olá André, tudo bem? Pelo que vi, o PR está falhando por que falta adicionar a `ProgressView` dentro da VStack:

```swift
        VStack {
             Image("EvercodeLogo")
                .resizable()
                .scaledToFit()
                .frame(width: 100, height: 100)

            Text("Welcome to Evercode!")
                .padding(16)
                .font(Font.title.weight(.semibold))

            ProgressView()
        }
```

**Review by rdgborges — COMMENTED** [review]
Para o PR passar, acredito que falta apenas uma pequena mudança:

**Comment by denise-techlead** [comment]
The unit tests did not pass. This means there is something wrong with your solution. Fix it and create a new commit on this branch to update this Pull Request. 👀

File: EvercodeApp/SplashView.swift [hunk 2]
```diff
@@ -2,7 +2,19 @@ import SwiftUI
...
+            Image("EvercodeLogo")
+                .resizable()
+                .scaledToFit()
```
Comment (rdgborges) [inline EvercodeApp/SplashView.swift]: Verifiquei que o `.scaledToFit()` altera um pouco o tamanho da imagem, ocasionando a quebra do teste que temos configurado para esta CodeStory. Neste caso, tente remover essa linha, criar um novo commit e atualizar o PR.

**Review by denise-techlead — APPROVED** [review]
Everything seems to be fine over here. Congratulations!

**Pull request merged**